assets/src/editor/events/event-editor.js

```diff
@@ -10,10 +10,20 @@ const editor = document.createElement( 'div' );
 editor.className = 'ee-editor-div';
 container.prepend( editor );
 const eventId = window.eeEditorEventId;
+const entityListView = window.eeEditorListView ?
+	window.eeEditorListView :
+	'grid';
+const entityListPerPage = window.eeEditorPerPage ?
+	window.eeEditorPerPage :
+	6;
 
 if ( eventId ) {
 	render(
-		<EventDatesAndTicketsMetabox eventId={ eventId } />,
+		<EventDatesAndTicketsMetabox
+			eventId={ eventId }
+			entityListView={ entityListView }
+			entityListPerPage={ entityListPerPage }
+		/>,
 		editor
```

Comment:
After doing the full review it looks like these are incoming options set server side.  Initially however, from the name I thought `entityListView` was an actual view component, but instead I see that it is indicating whether it is "grid" or "list".

Some thoughts I have:

- If you decide to keep this in, I think a change in variable name might be helpful.
- I'm wary of prematurely setting options this way as I think it as much as possible we should be careful about what props need to be passed in on initialization until we have a better idea of how we want to initialize. I'd be more inclined to have options loaded as a part of the _internal_  initialization process (as opposed to externally via the  so they could be either coming from browser localStorage (so there can be per user/browser defaults) OR defaults from the server (via api or global variable from the dom set by the server).
- Until we're ready to do that, I'd be inclined to simplify the admin option you are adding in this pull to just activating/deactivating the advanced editor and we can spend more time figuring out how we will initialize "defaults" in it's own pull.